Dockerfile

```diff
@@ -195,13 +197,20 @@ RUN fname="go-ipfs_${IPFS_VERSION}_${TARGETOS:-linux}-${TARGETARCH:-amd64}.tar.g
   tar -C ${tmpout} -xzf "${fname}" go-ipfs/ipfs && \
   mv ${tmpout}/go-ipfs/ipfs /out/bin/ && \
   echo "- IPFS: ${IPFS_VERSION}" >> /out/share/doc/nerdctl-full/README.md
+ARG TINI_VERSION
+RUN fname="tini-static-${TARGETARCH:-amd64}" && \
+  curl -o "${fname}" -fSL "https://github.com/krallin/tini/releases/download/${TINI_VERSION}/${fname}" && \
+  grep "${fname}" "/SHA256SUMS.d/tini-${TINI_VERSION}" | sha256sum -c && \
```

Comment:
@AkihiroSuda I don't add the sumfile to the repo. it will be downloaded every time

Reply:
The sum file has to be added to the repo to avoid supply chain attacks